### ISSUE TITLE
It change google search appearance

### ISSUE THREAD

**Issue by dehghani-mehdi** [containers-everywhere/contain-google#70]
Hi,

When I use this addon, I see this:
![Screenshot_1](https://user-images.githubusercontent.com/9152330/68667593-9ea50880-055b-11ea-867c-cc15b514ccab.png)

But without container or using Mozilla's addon, I see this:
![Screenshot_2](https://user-images.githubusercontent.com/9152330/68667624-b7152300-055b-11ea-8614-5c6c69a801cd.png)

Why?

**Comment by Perflyst** [comment]
Your first image is the new design from Google. This is nothing from the addon.

**Comment by dehghani-mehdi** [comment]
Why this happens only when I use your addon?

**Comment by Perflyst** [comment]
Maybe because of the different cookies? I cannot tell you exactly why at least it is the new google you will have soon everywhere by default

**Comment by dehghani-mehdi** [comment]
I tested in brand new profile but I got similar result, did you change location or something? in Opera when I turn on its VPN google shows me new looks but without VPN still old one.
At home I use your addon (actually I can not use Firefox without your addon) and it doesn't happens. I don't know why this happens.
For now I'm using your addon at work and using custom JS code I simulate old google. new appearance sux!
